### DISCUSSION TITLE
[Feature Request] Configuring MFA for Project

### DISCUSSION THREAD

**Discussion by AdamczykMaciej** [umbraco/Umbraco.Cloud.Issues#807] [Ideas]
### Issue description

Currently, it is not possible to enforce, configure MFA for members of a project. You have to invite them once again by inviting them to the Organization, then they need to accept the invitation and finally you can enforce MFA for them. 
That's not convenient. Instead it would be much better if we could configure that on the project scope level in Umbraco Cloud. Additionally, we don't want those users to have access to Umbraco Portal. By inviting them to the Organization they are able and change Company Information.

**Comment by ksuvec** [comment]
Hi @AdamczykMaciej 

Thank you for the suggestion. MFA is generally a difficult topic when there are multiple access layers involved, so we have started from the where it makes most sense. 

An idea could be to do as within github where each project could say something like members can only be added if they have MFA enabled?

Best
Kec

**Comment by AdamczykMaciej** [comment]
Hm. I think that won't work for content editors. They would need another account- Github account which might be confusing.

**Comment by celeritydesign** [comment]
Just adding support for this feature request. Currently we have to contact Umbraco first-line support on behalf of our users, and then wait for the reset request to get routed through to Umbraco DevOps.  This can take significant time and the editing team's expectation is that their admins should be able to do this for them.
